src/AppProvider.jsx

```diff
@@ -6,11 +6,11 @@ import { Router } from 'react-router-dom';
 
 import App from './App';
 import ErrorBoundary from './ErrorBoundary';
-import AuthenticationContext from './AuthenticationContext';
+import AppContext from './AppContext';
 
 const AppProvider = ({ store, children }) => (
```

Comment:
I'm good with the name _I think_, but this isn't strictly a provider is it?

Reply:
It's a rollup of a bunch of providers that provide things, so I don't see why it isn't one. :)

Reply:
Not sure there's a formal definition of "provider" beyond something that makes stuff available to the components it wraps.